### PR TITLE
Chore #9 reorganize file structure

### LINKED ISSUE #9: Move style files to one folder and call styles in root component
### Description

Right now every component explicitly calls its own style file. 

### Expected
Move all style files to one folder and import all of them into the root Component once.

### PR DESCRIPTION
**Pull request by Farhanisto**
**Resolves #9** 

**Description:** 
Reorganize file structure

**Fix:**
Used git mv command to re-organize the file structure.

### CODE REVIEW

**Comment by john555** [comment]
Could you rename the `Index.jsx` file to `index.jsx`. It does not contain a component, therefore it shouldn't start with an uppercase character.

File: src/Index.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```
Comment (john555) [inline src/Index.jsx]: Rename this file to `index.jsx`.

Comment (Farhanisto) [reply src/Index.jsx]: @john555 thanks for the pointer i have implemented the requested changes

**Comment by john555** [comment]
@joykare What do you think about this file structure?

**Comment by joykare** [comment]
going through it right now.

**Comment by Farhanisto** [comment]
@joykare we will change the containers and the components to go in the same tree level..... I think the reason we have so many files changed in one commit is cause we were re-organizing the whole codebase.

**Pull request merged**